res/values-az/strings.xml

```diff
@@ -328,7 +328,7 @@
     <string name="COMPLETE">"Tamamlandı"</string>
     <string name="INCOMPLETE">"Tamamlanmadı"</string>
     <string name="Win_a_FFA_Time_Game">"FFA (Vaxtlı) Oyununu qazanın"</string>
-    <string name="Win_a_CTF_Game">"CTF Oyununu Qazanın"</string>
+    <string name="Win_a_CTF_Game">"Bayraq qapma Oyununu Qazanın"</string>
```

Comment:
"CTF" wasn't been translated in azerbaijan language, however, if it's supposed to be translated, it must be with acronyms. Please return it back, or try finding an accurate acronym for it to translate.